src/aws-lambda/personalize-pre-create-campaigns/personalize-pre-create-campaigns.py

```diff
@@ -256,6 +264,23 @@
                      'filename': offer_interactions_filename}}
 
 
+def get_all_product_categories():
```

Comment:
Is this still being used with the new filter approach?

Reply:
Deleted in the latest changes.